l10n_ch_payment_slip/wizard/bvr_import.py

```diff
@@ -206,7 +206,7 @@ def _prepare_line_vals(self, statement, record):
         )
         if line_ids:
```

Comment:
In fact `line_ids` is not named properly as it is a record set.
Plus can we have multiple lines returned by search?
If yes and we still want to manage only one, we can add `limit=1` on search

Please add `limit=1` and rename `line_ids` to `line` and remove `line = line_ids`